### PR TITLE
Prepare azure-sdk-bom For Release

### PR DESCRIPTION
**Pull request by alzimmermsft**
This PR updates the versions used `azure-sdk-bom` in preparation for release.

### CODE REVIEW

File: sdk/boms/azure-sdk-bom/pom.xml [hunk 1]
```diff
@@ -55,16 +60,36 @@
...
+      </dependency>
+      <dependency>
+        <groupId>com.azure</groupId>
+        <artifactId>azure-messaging-eventhubs-checkpointstore-blob</artifactId>
+        <version>1.0.2</version>
+        <exclusions>
+          <exclusion>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-storage-blob</artifactId>
+          </exclusion>
```
Comment (alzimmermsft) [inline sdk/boms/azure-sdk-bom/pom.xml]: Need to validate what a dependency exclusion does in a BOM.

File: sdk/boms/azure-sdk-bom/ci.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      - hotfix/*
+      - release/*
+  paths:
+    include:
+      - sdk/boms/azure-sdk-bom/
+
+stages:
+  - template: ../../eng/pipelines/azure-sdk-bom.yml
+    parameters:
+      ServiceDirectory: appconfiguration
```
Comment (weshaggard) [inline sdk/boms/azure-sdk-bom/ci.yml]: This doesn't look correct.

Comment (weshaggard) [reply sdk/boms/azure-sdk-bom/ci.yml]: Looks like you can just delete this entire parameter for boms.

File: eng/pipelines/azure-sdk-bom.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+trigger: none
+pr: none
+
+jobs:
+  - job: Build and Validate
+    pool:
+      vmImage: 'ubuntu-16.04'
+
+    steps:
```
Comment (weshaggard) [inline eng/pipelines/azure-sdk-bom.yml]: What about hooking up the signing and releasing parts of the pipeline? Do you intend to do that manually for now?

fyi @mitchdenny

Comment (alzimmermsft) [reply eng/pipelines/azure-sdk-bom.yml]: I'll want to add those, should have some follow-up changes to add this and make it generic for other potential BOMs we'll be releasing out of the repo.

**Pull request merged**